docs/source/changelog.rst

```diff
@@ -9,21 +9,58 @@ Changes in nbconvert
 
 The following authors and reviewers contributed the changes for this release -- Thanks you all!
 
-- Philip Austin
-- Maarten Breddels
-- David Brochart
-- Matthias Bussonnier
-- Sylvain Corlay
-- Matthias Geier
-- Tyler Makaro
-- Matthew Seal
-- Jeremy Tuloup
+* admin
+* Ayaz Salikhov
+* bnables
+* Bo
+* David Brochart
+* David Cortés
+* Eric Wieser
+* Florian Rathgeber
+* Ian Allison
+* James Wilshaw
+* Jeremy Tuloup
+* Joel Ostblom
+* joelostblom
```

Comment:
You can remove this username entry if you want, my full name is in the entry above. I'm good either way and cool that I am listed here, thank you! =D